<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListClientsDialog</class>
 <widget class="QDialog" name="ListClientsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clients and servers</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/qutescoop.png</normaloff>:/icons/qutescoop.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabSpacing">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageClients">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>732</width>
        <height>418</height>
       </rect>
      </property>
      <attribute name="label">
       <string>C&amp;lients</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="editFilter">
           <property name="toolTip">
            <string>Wildcards (*, ?) and those powerful &quot;Regular Expressions&quot; (search for them on the internet) allowed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
           <property name="buddy">
            <cstring>editFilter</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="boxResults">
         <property name="title">
          <string>Results</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeView" name="treeListClients">
            <property name="toolTip">
             <string>Lists all connected Clients for the selected network</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageServers">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>125</width>
        <height>131</height>
       </rect>
      </property>
      <attribute name="label">
       <string>&amp;Servers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pbPingServers">
           <property name="toolTip">
            <string>Find the &quot;ping&quot; to the server, that is the time (in milliseconds) a packet needs to travel between you and the server. This is also called &quot;network lag&quot;</string>
           </property>
           <property name="text">
            <string>&amp;ping servers</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="serversTable">
         <property name="toolTip">
          <string>Lists all servers that have been announced in Whazzup</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageVoiceServers">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>125</width>
        <height>131</height>
       </rect>
      </property>
      <attribute name="label">
       <string>&amp;Voice servers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pbPingVoiceServers">
           <property name="toolTip">
            <string>Find the &quot;ping&quot; to the server, that is the time (in milliseconds) a packet needs to travel between you and the server. This is also called &quot;network lag&quot;</string>
           </property>
           <property name="text">
            <string>&amp;ping servers</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="voiceServersTable">
         <property name="toolTip">
          <string>Lists all Voice Servers that have been announced in Whazzup.
Note: Voice Servers are not choosable for you as a pilot. ATC selects the server and you will join on his/her server.</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblStatusInfo">
       <property name="text">
        <string>waiting for data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>ListClientsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
